src/main/java/org/terasology/launcher/gui/javafx/SettingsController.java

```diff
@@ -19,30 +19,17 @@
 import javafx.event.ActionEvent;
 import javafx.fxml.FXML;
 import javafx.scene.Node;
-import javafx.scene.control.Button;
-import javafx.scene.control.CheckBox;
-import javafx.scene.control.ComboBox;
-import javafx.scene.control.Label;
-import javafx.scene.control.ListCell;
-import javafx.scene.control.Tab;
-import javafx.scene.control.TextField;
+import javafx.scene.control.*;
```

Comment:
please don't use star imports. you can set the corresponding setting in intellij to a very high number (e.g., 9999) to avoid auto-formatting.